nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/impl/AbstractDocumentModelWriter.java

```diff
@@ -207,6 +212,15 @@ protected boolean loadFacetsInfo(DocumentModel docModel, Document doc) {
         while (facets.hasNext()) {
             Element element = facets.next();
             String facet = element.getTextTrim();
+
+            SchemaManager schemaManager = Framework.getService(SchemaManager.class);
+            CompositeType facetType = schemaManager.getFacet(facet);
+
+            if (facetType == null) {
+                log.warn("The facet: " + facet + " does not exist, it will be ignored.");
+                continue;
```

Comment:
👍 we should at least log the document type and useful information like that on warn level, and more information on debug level as @atchertchian suggests.